### PR TITLE
APP-182: [Objective-C] Basic level Apps: Endpoint profiles and grouping

### PR DESCRIPTION
**Pull request by am1kk**
http://jira.kaaproject.org/browse/APP-182

### CODE REVIEW

**Comment by Slash32** [comment]
Please re-base the PR on top of develop avoiding commits related to APP-93.

File: profilingdemo/source/objc/ProfilingDemo/ViewController.m [hunk 1]
```diff
@@ -107,7 +107,7 @@ - (void)displayConfiguration {
     [self addLogWithText:[NSString stringWithFormat:@"Audio support enabled: %@, video support enabled: %@, vibro support enabled: %@", configuration.audioSubscriptionActive ? @"yes" : @"no", configuration.videoSubscriptionActive ? @"yes" : @"no", configuration.vibroSubscriptionActive ? @"yes" : @"no"]];
 }
 
-- (void) addLogWithText:(NSString *) text {
+- (void)addLogWithText:(NSString *) text {
```
Comment (hamsternik) [inline profilingdemo/source/objc/ProfilingDemo/ViewController.m:110]: And remove space between _(NSString *)_ and _text_ argument :)

**Comment by vilimets** [comment]
LGTM

**Comment by Slash32** [comment]
LGTM

**Pull request merged**